### PR TITLE
Adding async private route example

### PR DESCRIPTION
**Pull request by oldtinroof**

### CODE REVIEW

**Comment by oldtinroof** [comment]
@mefechoel I have no idea why the linting is failing on one of the testApp files - I didn't open that file; I've run the lint script locally and get no issues (after my second commit on this PR where I remembered to run linting!).

**Comment by mefechoel** [comment]
Thanks so much for the detailed example and description!
I'll check out the file that is causing the failure and merge this in the next couple of days.

**Review by mefechoel — CHANGES_REQUESTED** [review]

File: test/testApp/src/Main.svelte [hunk 1]
```diff
@@ -26,8 +26,7 @@
 				"data-has-attr": true,
```
[the remark this answers is not in the document]
Comment (oldtinroof) [reply test/testApp/src/Main.svelte]: How strange - I didn't open this file, I guess it got updated when I ran the lint:fix command. I wonder if I'm running a different version of Prettier or a global config got involved... Anyway, reverted now 👍

File: example/private-routes-async/src/App.svelte [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			<h3>Home</h3>
+			<p>Home sweet home...</p>
+		</Route>
+
+		<Route path="about">
+			<h3>About</h3>
+			<p>That's what it's all about!</p>
+		</Route>
+
+		<PrivateRoute path="profile" let:location>
```
[the remark this answers is not in the document]
Comment (oldtinroof) [reply example/private-routes-async/src/App.svelte]: Sure thing 👍

**Pull request merged**